docs/strategy-advanced.md

```diff
@@ -8,11 +8,111 @@ If you're just getting started, please be familiar with the methods described in
 !!! Note
     All callback methods described below should only be implemented in a strategy if they are actually used.
 
+!!! Tip
+    You can get a strategy template containing all below methods by running `freqtrade new-strategy --strategy MyAwesomeStrategy --template advanced`
+
+## Custom stoploss
+
+A stoploss can only ever move upwards - so if you set it to an absolute profit of 2%, you can never move it below this price.
+Also, the traditional `stoploss` value serves as an absolute lower level and will be instated as the initial stoploss.
+
+The usage of the custom stoploss method must be enabled by setting `use_custom_stoploss=True` on the strategy object.
+The method must return a stoploss value (float / number) with a relative ratio below the current price.
+E.g. `current_profit = 0.05` (5% profit) - stoploss returns `0.02` - then you "locked in" a profit of 3% (`0.05 - 0.02 = 0.03`).
+
+``` python
+    use_custom_stoploss = True
+
+    def custom_stoploss(self, pair: str, trade: Trade, current_time: datetime, current_rate: float,
+                        current_profit: float, **kwargs) -> float:
+        # TODO: Add full docstring here
+        return 0.04
+```
+
```

Comment:
Could the example (the `E.g....` part) maybe be moved to after the example?

I'm reviewing the documentation before the code and so far it's a _bit_ hard for me to follow what the example is saying. I imagine it might be clearer if there's the context of the example 0.04 value in the strategy > `custom_stoploss`

Reply:
i suspect it will depend on the background - if you're more familiar with the code, then having the code first makes it simpler - if you wanna understand the logic first, then look at the code how it's done, then this way might be better...

i will need to add the docstring to the docs too, anyway ... so i'll think about how to make this clearer :+1:

Reply:
Yeah, it makes sense to be friendly to someone not familiar with code. I guess I'm just looking for clarification on the example. And I wasn't sure what relationship - if any - there is between the example and the code snippet.

Reply:
well in this case - there isn't :D 

the "real" examples start below this point ... this was there just to demonstrate the function header more than anything... i'll change this slightly